recommender/recommender.py

```diff
@@ -1060,4 +1060,24 @@ def strip_and_clean_html_elements(data):
     """
     Clean an HTML elements and return it
     """
-    return bleach.clean(data, strip=True)
+    return bleach.clean(data, tags=[], strip=True)
+
+def strip_and_clean_url(data):
+    """
+    Clean an URL elements and return it
```

Comment:
what is an example of `URL elements`? I think the example you gave for linkify is nice. Can we have something similar here?